README.md

```diff
@@ -1,6 +1,31 @@
-# questland-public-api
-This is a public API for Questland
+# Questland API
+This is a public API for Questland inspired by Mandalorian007. 
+Current public url: http://34.217.43.20:8080/swagger-ui.html#/
 
-TODO add real docs :)
+# Purpose
+Help the community build and grow. I offer informative videos on my
+YouTube to help players catch up with ease to enjoy the game and
+published a "handbook" that offers guides on everything you need
+to know! I have also created a questland discord bot that is backed by
+the questland public api. To earn exclusive perks become a Patreon by pledging
+$3 or more, though it's not required, but greatly appreciated.
 
-Current public url: http://34.217.43.20:8080/swagger-ui.html#/
\ No newline at end of file
+# Youtube
+My youtube provides the basics such as Gear, a variety of builds,
+and the Battle Events.
+https://www.youtube.com/channel/UCLHjCEBoRj-PGCPvmWzQXMQ/videos
+
+# Questland Handbook
+This website provides extensive guides and in depth information on everything you need to know about
+Questland.
+https://questland-handbook.cfapps.io/
+
+# Discord Bot
+https://github.com/Mandalorian007/questland-discord-bot
```

Comment:
Let's also add the install link so an admin can add QL Bot to their server:
https://discordapp.com/oauth2/authorize?client_id=675765765395316740&scope=bot